src/script/ml.ts

```diff
@@ -64,7 +61,7 @@ function createModel(): LayersModel {
   return model;
 }
 
-export async function trainModel(): Promise<boolean> {
+export async function trainModel() {
```

Comment:
We may as well keep the type and change it to how it use to be.
```suggestion
export async function trainModel(): Promise<void> {
```

Also need to change line 74 from `return false;` to `return;`